tests/end_to_end/test_emotet/test_emotet.py

```diff
@@ -0,0 +1,70 @@
+import json
+import os
+import re
+import pytest
+from tempfile import gettempdir
+
+from wazuh_testing import end_to_end as e2e
+from wazuh_testing import event_monitor as evm
+from wazuh_testing.tools import configuration as config
+
+
+alerts_json = os.path.join(gettempdir(), 'alerts.json')
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+test_cases_file_path = os.path.join(test_data_path, 'test_cases', 'cases_emotet.yaml')
+configuration_playbooks = ['configuration.yaml']
+emotet_file_path = os.path.join(test_data_path, 'emotet_file', 'trigger-emotet.exe')
+sysmon_config = os.path.join(test_data_path, 'sysmon_config', 'sysconfig.xml')
+configuration_extra_vars = {'emotet_file': emotet_file_path, 'sysmon_config': sysmon_config}
+
```

Comment:
```suggestion
```

Reply:
Done in a0a100ae7101eb33a0505647449094d4f1a34646